charts/xwiki/values.yaml

```diff
@@ -159,9 +159,27 @@ externalDB:
 ## To use external solr enable solr, provide host and port
 ##
 solr:
-  enabled: true
-  host: localhost
-  port: 8983
+  enabled: true #If set to false mount an internal Solr instance
+  replicaCount: 1
+  image: gridexx/solr:1001
```

Comment:
I should have put a comment. Here the image is not the final one. We should wait for Vincent answer and the official XWiki image to replace it when it will be okay.